buildSrc/src/main/kotlin/extensions.kt

```diff
@@ -0,0 +1 @@
+const val buildReceiptName = "build-receipt.properties"
```

Comment:
This value is duplicated in  buildSrc/src/main/kotlin/profiler.versioning.gradle.kts. Can it be reused?

Reply:
Well spotted! I removed it in `profiler.versioning.gradle.kts`.